experiments/standalone/Bucket/global_bucket_function.jl

```diff
@@ -164,6 +169,38 @@ updatefunc = ClimaLand.make_update_drivers(bucket_atmos, bucket_rad)
 driver_cb = ClimaLand.DriverUpdateCallback(updateat, updatefunc)
 cb = SciMLBase.CallbackSet(driver_cb, saving_cb)
 
+
+
+# ClimaDiagnostics
+output_dir = "output/"
+ClimaUtilities.OutputPathGenerator.generate_output_path(output_dir)
+
+space = bucket_domain.space.subsurface
+
+nc_writer = ClimaDiagnostics.Writers.NetCDFWriter(space, output_dir)
+
+diags =
+    ClimaLand.CLD.default_diagnostics(model, 1.0; output_writer = nc_writer)
```

Comment:
This 1.0 is unclear

Reply:
I agree. I just put a value for `t_start`, but maybe we don't need it? It's an optional arg in `EveryDtSchedule(period; t_start)` called in `common_diagnostics`

Reply:
Yes, `t_start` is good!

Reply:
Let's come back to this too - I am not sure I followed, do we keep the 1.0? Or adjust something?